w3d_vgui.lua

```diff
@@ -80,26 +80,37 @@ end
 -- Check if the mouse is hovering something on the screen
 function w3d.IsHovered(x, y, w, h)
 
-    local iMouseX, iMouseY = w3d.tCache.iMouseX, w3d.tCache.iMouseY
-    
+    local iMouseX, iMouseY = w3d.GetMousePos()
```

Comment:
Why would you use this instead of the cache value? It's more optimized to directly fetch the variable than passing by a function call to get them. And I were using it for a good reason: with the function, the fallback value is 0 in case the variables are not defined -> that's an unwanted behaviour since they could be detected as hovered while the values are not even defined.